macros/showMore.njk

```diff
@@ -0,0 +1,13 @@
+{% macro showMore(label, content) %}
+  <div class="show-more">
+    <div class="bold-small">{{ t(label) }}</div>
+    {% if longText.length > 400 %}
```

Comment:
Where the `longText` variable is defined?

Reply:
forgot to change it when I pulled it out to the common-frontend. Updated now